runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerPlugin.java

```diff
@@ -144,9 +152,12 @@ public void onItemContainerChanged(final ItemContainerChanged event)
 	@Subscribe
 	public void onGameTick(GameTick tick)
 	{
-		if (config.prayerFlickHelper())
+		startOfLastTick = Instant.now();
```

Comment:
Move start of last tick and prayersActive inside the if (!config.prayerFlickLocation().equals(PrayerFlickLocation.NONE))

Reply:
prayersActive is also used by the PrayerBar overlay to display the base prayer bar.

Reply:
ah alright. then just startOfLastTick I guess or alter the condition to also check if base prayer bar is enabled

Reply:
Would it not be best to leave this outside of the condition otherwise it would need to be placed in both the above condition and also in `if (config.showPrayerBar())`

Reply:
well alright i guess